pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -28,6 +28,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/apiserver/pkg/util/webhook"
 	"k8s.io/kubernetes/pkg/apis/admissionregistration"
+	v1 "k8s.io/kubernetes/pkg/apis/admissionregistration/v1"
```

Comment:
nit: I think v1 is an alias for `core/v1`, we should use a different alias here.

Reply:
fixed